cypress/integration/examples/add_conflicting_product.spec.js

```diff
@@ -0,0 +1,48 @@
+/// <reference types="cypress" />
+
+import Chance from "chance";
+const chance = new Chance();
+
+const randomCheckbox = chance.integer({ min: 0, max: 6 });
```

Comment:
It's fine to use a module to generate random numbers. As an alternative, I would create a function to generate a number between `min` and `max`:
You can search for `js random between two numbers`.